container.fc

```diff
@@ -20,11 +20,14 @@
 /usr/s?bin/buildkitd.*		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/s?bin/buildkitd.*	--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 
+/usr/s?bin/incus-.*		--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
can we group the incus changes?

Reply:
I think it's probably worth reorganizing this file to group all the different related ones in a separate PR. I tried to keep the Incus/LXD ones together in this PR, though.